meeting-notes/2018/201Y8-11-12--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,206 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 November 12, 2018
+- **Lead:** @eingenito
+- **Notetaker:**
+- **Attendees:**
+  - _attendee names..._
+  - @stebalien
+  - @hannahhoward
+  - @michaelavila
+  - @alanshaw
+  - @warpfork
+  - @momack2
+  - @daviddias
+  
+- **Recording:** _add link to recording once it's online_
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+  - Mid-Q OKRs
+
+
+## Notes
+
+### Week Update
+
+@name
+  - Done:
+  - Blocked:
+  - Next:
+
+@marten-seemann
+ - Done:
+ 	- attended IETF 103 meeting & hackathon in Bangkok
+  	- I discovered a packet injection attack on the QUIC handshake, held a presentation about this in the QUIC WG meeting and proposed a fix, which was accepted
```

Comment:
This is super cool! Can we get a tiny blog post out of it? Perhaps a mention on the IPFS Newsletter? //cc @renrutnnej